.github/workflows/hlint.yml

```diff
@@ -12,11 +12,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: haskell/actions/hlint-setup@v2
-        with:
-          version: '3.4'
-      - uses: haskell/actions/hlint-run@v1
-        with:
-          hlint-bin: "hlint --hint=./.github/.hlint.yaml"
-          fail-on: status
```

Comment:
This is important! This is how it worked in the first place! I think maybe we should instead do something like
```
nix build .#hlint
...
hlint-bin: "./result/bin/hlint"
```
or something

Reply:
We do want to use `hlint-run`. It's doesn't just run hlint, it turns the output into github checks output.